.github/workflows/build.yaml

```diff
@@ -158,6 +158,18 @@ jobs:
         name: "update__${{ github.ref_name }}_(${{ steps.vars.outputs.sha_short }})"
         path: ./dist/*
 
+    - name: Store generated files in cache
+      uses: actions/cache@v3
+      with:
+        path: |
+          ./code/.pio/build/esp32cam/firmware.bin
+          ./code/.pio/build/esp32cam/partitions.bin
+          ./code/.pio/build/esp32cam/bootloader.bin
+          ./sd-card/html/version.txt
+          ./dist
+        key: ${{ github.run_number }}-pack-for-OTA-v2
+
+
 
 
 #########################################################################################
```

Comment:
`pack-for-fresh-install` doesn't have any use anymore, right?

Reply:
I did't saw it. Will change the build structure for cleaner reading.